src/Sulu/Bundle/AdminBundle/Resources/js/containers/index.js

```diff
@@ -63,7 +63,7 @@ export {
     MultiListOverlay,
     MultiSelection,
     InfiniteLoadingStrategy,
-    PaginatedLoadingStrategy,
+    DefaultLoadingStrategy,
```

Comment:
```suggestion
    DefaultLoadingStrategy,
    PaginatedLoadingStrategy,
```
we still need to export the `PaginatedLoadingStrategy` for backwards compatibility 🙂